_topics/en/replace-by-fee.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Replace-by-Fee
+
+aliases:
+  - BIP125
+  - Opt-in Replace-by-Fee
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Fee Management
+  - Transaction Relay Policy
+  - Mining
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Replace by Fee (RBF)** is the ability to replace an unconfirmed
+  transaction in a mempool with a different transaction that spends at
+  least one of the same inputs and which pays a higher transaction fee.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  RBF is a policy for mining and relaying transactions, so it's
```

Comment:
s/mining and relaying/relaying and mining/ since relaying comes first (and happens more often than mining)